remappings.txt

```diff
@@ -12,3 +12,33 @@ mento-core-2.5.0/=lib/mento-core-2.5.0/contracts/
 merkle-distributor/=lib/merkle-distributor/contracts/
 mento-core-2.6.0-oz/=lib/mento-core-2.6.0/lib/openzeppelin-contracts-next
 mento-core-2.6.0/=lib/mento-core-2.6.0/contracts/
+openzeppelin-solidity/=lib/mento-core-2.0.0/lib/openzeppelin-contracts/
```

Comment:
some of these seem to be duplicated and not very relevant (i.e. `chainlink-brownie-contracts`, foundry-chainlink-toolkit`) maybe remove this file from the changes or just leave the ones that are needed?

Reply:
@nvtaveras So essentially I ran the `forge remappings >> remappings.txt` for this.

I'd assume that if an entry is added here that there is a potential for it to be needed later with little affect on performance. 

That being said I am unsure of how likely such a situation is